test/Chainweb/Test/MultiNode.hs

```diff
@@ -283,6 +288,62 @@ runNodesForSeconds loglevel write baseConf n (Seconds seconds) rdb pactDbDir inn
     void $ timeout (int seconds * 1_000_000)
         $ runNodes loglevel write baseConf n rdb pactDbDir inner
 
+-- | Run nodes
+--   Each node creates blocks
+--   We wait until they've made a sufficient amount of blocks
+--   We stop the nodes
+--   We open sqlite connections to some of the database dirs and compact them
+--   We restart all nodes with the same database dirs
+--   We observe that they can make progress
+compactAndResumeTest :: ()
+  => LogLevel
+  -> ChainwebVersion
+  -> Natural
+  -> TestTree
+compactAndResumeTest logLevel v n =
+  let
+    name = "compact-resume"
+  in
+  after AllFinish "ConsensusNetwork" $ testCaseSteps name $ \step ->
+  withTempRocksDb "compact-resume-test-rocks" $ \rdb ->
+  withSystemTempDirectory "compact-resume-test-pact" $ \pactDbDir -> do
+    let tastyLog = step . T.unpack
+    let logFun = tastyLog
```

Comment:
Was copying from below (`replayTest`). I can see the argument for it, because it differentiates between what is a test log msg from us the programmer in the test itself, vs something transitively called by our code.

However, I also am somewhat inclined to remove it.